docs/pages/router/reference/api-routes.mdx

```diff
@@ -196,6 +198,163 @@ export async function GET(request: Request, { post }: Record<string, string>) {
 
 Making requests with an undefined method will automatically return `405: Method not allowed`. If an error is thrown during the request, it will automatically return `500: Internal server error`.
 
+## Runtime API
+
+> **important** The server runtime API and `expo-server` are available in SDK 54 and later, and require a deployed server for production use.
+
+You can use the `expo-server` package to use several utilities and code patterns that work in any server-side Expo code. This includes utilities to get request metadata, for scheduling tasks, and for error handling.
+
+<Terminal cmd={['$ npx expo install expo-server']} />
+
+Using `expo-server` is not limited to API routes and it can be used in any other server code as well, for example in [server middleware](/router/reference/middleware/).
```

Comment:
```suggestion
Using `expo-server` is not limited to API routes and it can be used in any other server code as well, for example, in [server middleware](/router/reference/middleware/).
```